sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3950,6 +3950,14 @@ object SQLConf {
     .checkValues(ErrorMessageFormat.values.map(_.toString))
     .createWithDefault(ErrorMessageFormat.PRETTY.toString)
 
+  val SPARK_MASKING_ALGO = buildConf("spark.sql.masking.algo")
```

Comment:
how about `spark.sql.masking.algo.fipsModeEnabled`

Reply:
I was trying to follow a similar property value approach as in [HiveConf](https://github.com/apache/hive/blob/master/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java#L4487) ,  where SHA algorithm type is used to decide whether FIPS Mode is enabled or not.
Do you still recommend to change it to boolean property `spark.sql.masking.algo.fipsModeEnabled` ?

Reply:
I think we usually use the same name for the config entry and helper function. Let's see other ones' opinions.

Reply:
can't we let the function take one more parameter? It's confusing to use a config to control the behavior of a specific function.

Reply:
Yes, we can add a third boolean parameter to control the fips mode. I'll update the code